### DISCUSSION TITLE
Looking for a HiBob adapter and happy to pay for it

### DISCUSSION THREAD

**Discussion by aidanlister** [airbytehq/airbyte#39356] [New Connector Request]
Hi,

I'm looking for a HiBob adapter and happy to pay for it.

HiBob is one of the most popular HRIS systems in the world,
https://www.hibob.com/

They have a lovely API and there's already an integration with FiveTrans if you wanted to see how it works.

We're after employee data: name, email, business entity, business division, current salary.

**Comment by natikgadzhi** [comment]
Hey @aidanlister, I'm testing something internally, and I'd love to help. Do you happen to have a test account / token I can use to test the connector?

**Reply by aidanlister** [reply]
Sure can you send me an email? aidan@uptickhq.com

**Comment by natikgadzhi** [comment]
Hey @aidanlister, we made and shipped a hibob connector — ping us if you have any problems with it. 

https://docs.airbyte.com/integrations/sources/hibob
